EXILED/Exiled.API/Features/Toys/AdminToy.cs

```diff
@@ -33,9 +40,14 @@ internal AdminToy(AdminToyBase toyAdminToyBase, AdminToyType type)
             AdminToyBase = toyAdminToyBase;
             ToyType = type;
 
-            Map.ToysValue.Add(this);
+            BaseToAdminToy.Add(toyAdminToyBase, this);
         }
 
+        /// <summary>
+        /// Gets a list of all <see cref="AdminToy"/>'s on the server.
+        /// </summary>
+        public static IEnumerable<AdminToy> List => BaseToAdminToy.Values;
```

Comment:
can be `IReadOnlyCollection`

```suggestion
        public static IReadOnlyCollection<AdminToy> List => BaseToAdminToy.Values;
```